src/ORM/DataObject.php

```diff
@@ -940,6 +949,50 @@ public function i18n_plural_name()
         return _t(static::class . '.PLURALNAME', $this->plural_name());
     }
 
+    /**
+     * Get description for this class
+     * @return null|string
+     */
+    public function classDescription()
```

Comment:
```suggestion
    public function getClassDescription()
```
And update SiteTree::classDescription() to call $this->getClassDescription()

Reply:
Conversation about this will be held in https://github.com/silverstripe/silverstripe-framework/pull/11461#discussion_r1839344166